feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailState.kt

```diff
@@ -1,13 +1,17 @@
 package io.filmtime.feature.movie.detail
 
 import io.filmtime.core.ui.common.UiMessage
+import io.filmtime.data.model.MovieCollection
 import io.filmtime.data.model.Ratings
 import io.filmtime.data.model.StreamInfo
 import io.filmtime.data.model.VideoDetail
 
 data class MovieDetailState(
   val isLoading: Boolean = false,
   val isBookmarked: Boolean = false,
+  val hasCollection: Boolean = false,
```

Comment:
We do not need a redundant `hasCollection` field. Checking nubility with `videoDetail.collectionId ` works for us